src/pages/customers/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../../styles/common.css" />
+    <script defer src="/script/main.js"></script>
+    <title>Billbook | Customers</title>
+  </head>
+
+  <body class="dashboard">
+    <nav class="nav-bar" id="nav-bar">
+      <ul>
+        <a href="#">
+          <li class="nav-item nav-item--active"><i class="material-icons">person</i> Customers</li>
+        </a>
+        <a href="/pages/items">
+          <li class="nav-item"><i class="material-icons">star</i>Items</li>
+        </a>
+        <a href="/pages/invoice">
+          <li class="nav-item"><i class="material-icons">description</i>Invoices</li>
+        </a>
+      </ul>
+    </nav>
+
+    <nav class="bottom-nav">
+      <ul>
+        <li class="bottom-nav-item bottom-nav-item--active">
+          <a href="#"><i class="material-icons">person</i></a>
+        </li>
+        <li class="bottom-nav-item">
+          <a href="/pages/items"><i class="material-icons">star</i></a>
+        </li>
+        <li class="bottom-nav-item">
+          <a href="/pages/invoice"><i class="material-icons">description</i></a>
+        </li>
+        <li class="bottom-nav-item">
+          <button onclick="showNavBar(event)"><i class="material-icons">menu</i></button>
+        </li>
+      </ul>
+    </nav>
+
+    <main class="content">
+      <header class="content__header">
+        <h1>Customers</h1>
+        <a class="btn" href="/pages/customers/create.html">
+          <i class="material-icons btn__icon btn__icon--left">add</i>Add Customers
+        </a>
+      </header>
+
+      <div class="scrollable-table">
```

Comment:
Nit:
can it be just `scrollable`? there is nothing specific for table here. if this is wrapped around any other elements, it'll make that scrollable too.